docs/modules/provisioning-app/pages/index.adoc

```diff
@@ -14,6 +14,7 @@ It delegates the tasks to create / update resources to several services such as
  ** a Confluence Space (name based on project `key`)
  ** if OpenShift was selected - a Bitbucket Project (name based on project `key`). Within this project two default repositories are created: `<project-key>-oc-config-artifacts` for all OpenShift YAML resources used during export/import as well as `<project-key>-design<project>` for any design artifacts (e.g. sketches)
  ** if OpenShift was selected - the required Openshift projects named `<project-key>-dev`, `<project-key>-test` and `<project-key>-cd`. Internally those are created and configured using a Jenkins pipeline.
+ ** If you've set up your ODS infrastructure properly, then you've probably already filled up the cd user fields here in our ods-core environment file link:https://github.com/opendevstack/ods-core/blob/master/configuration-sample/ods-core.env.sample#L137[sample]. Once you create a new project, it adds this cd user credential information you provided to the cd component of your project. This is the technical user that will communicate with the ODS components of your fork. In order for your project to work in sync with ODS components, it needs to have read permissions over two ODS repositories: `ods-jenkins-shared-library` and `ods-quickstarters`. The read permission for shared library is needed so that the library can checkout your changes and deploy them into designated Openshift environment. As for the ladder ODS repository, when you generate a new component it needs to access the chosen quickstarter repository and execute it's Jenkinsfile. If you don't want a single cd user to have read access to all your projects, you can add a new project specific cd user that works in sync with ODS components. More information can be found xref:cd-user.adoc[here].
```

Comment:
> Once you create a new project, it adds this cd user credential information you provided to the cd component of your project

Only true if "project specific user" is not selected - which it should normally be.

Reply:
Why not make project specific the default then?